### PR TITLE
Added creator/editor read-only fields in edit resource properties modal 

### PR DESCRIPTION
**Pull request by Igi-ID**
## Description
This PR introduces the creator and editor fields inside the "Edit Properties" modal for all resources. 

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [x] Feature

## Issue

**What is the current behavior?**
The creator and editor fields are not visible.
[#9550](https://github.com/geosolutions-it/MapStore2/issues/9550)

**What is the new behavior?**
The creator and editor fields are visible inside the modal. The editor field correctly updates in scenarios where the editor !== creator and the change is persisted and visualized, i.e. the last editor is shown.

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

**Review by offtherailz — CHANGES_REQUESTED** [review]
- [x] The admin only requirement is not satisfied. We should make it configurable only it is **really really** simple, by default visible to admin only. 

- [ ] Note: the lines do not appear initially. When you save a resource, both creator and editor are persisted on the db. This may appear confusing a buggy, even if "by design". 
I suggest to always apply the migration scripts (and so report it in migration guidelines for @Igi-ID I'll do it on our servers) . This will always show creator information.



In particular the migration guidelines should explain that if `mapstore` automatically updates the table columns, that is the default mode ( the mode is `update`, link also [this](https://docs.mapstore.geosolutionsgroup.com/en/latest/developer-guide/database-setup/#database-creation-mode) section in your migration guideline section) but if they want the creators to be populated, they will have to execute the second part of the script.
The script have to be execututed whole if the mode is "validate" 

refer scripts for proeper database

**Comment by Igi-ID** [comment]
> * [ ]  The admin only requirement is not satisfied. We should make it configurable only it is **really really** simple, by default visible to admin only.
> * [ ]  Note: the lines do not appear initially. When you save a resource, both creator and editor are persisted on the db. This may appear confusing a buggy, even if "by design".
>   I suggest to always apply the migration scripts (and so report it in migration guidelines for @Igi-ID I'll do it on our servers) . This will always show creator information.
> 
> In particular the migration guidelines should explain that if `mapstore` automatically updates the table columns, that is the default mode ( the mode is `update`, link also [this](https://docs.mapstore.geosolutionsgroup.com/en/latest/developer-guide/database-setup/#database-creation-mode) section in your migration guideline section) but if they want the creators to be populated, they will have to execute the second part of the script. The script have to be execututed whole if the mode is "validate"
> 
> refer scripts for proeper database

Requested changes included in https://github.com/geosolutions-it/MapStore2/pull/9968/commits/43c2447b04a10fb36bfc73e6cf531f5194c3e5b0

File: docs/developer-guide/mapstore-migration-guide.md [hunk 1]
```diff
@@ -22,6 +22,37 @@ This is a list of things to check if you want to update from a previous version
...
+
+    [postgresql migration script 2.0.0 to 2.1.0](https://github.com/geosolutions-it/geostore/blob/master/doc/sql/migration/postgresql/postgresql-migration-from-v.2.0.0-to-v2.1.0.sql)
+
+    [h2 migration script 2.0.0 to 2.1.0](https://github.com/geosolutions-it/geostore/blob/master/doc/sql/migration/h2/h2-migration-from-v.2.0.0-to-v2.1.0.sql)
+
+    [oracle migration script 2.0.0 to 2.1.0](https://github.com/geosolutions-it/geostore/blob/master/doc/sql/migration/oracle/oracle-migration-from-v.2.0.0-to-v2.1.0.sql)
+
+- In case of a `validate` database creation option:
+  - The table columns will not be updated.
+  - You have to manually execute the whole script referenced above, depending on your choice of database type.
```
Comment (offtherailz) [inline docs/developer-guide/mapstore-migration-guide.md:54]:
```suggestion
The resource metadata has been recently extented to include  information about resource creator and editor and to provide the advertises/unadvertised resource functionalities.
If your installation has the [database creation mode](https://docs.mapstore.geosolutionsgroup.com/en/latest/developer-guide/database-setup/#database-creation-mode) set to `update` (the default), the columns will be added automatically and you do not have to do any action. If it is set to `validate` instead you will have to run the update scripts.

*In any case*, the update scripts contain also a part to populate the `creator` column, that can not be applied automatically with  [database creation mode](https://docs.mapstore.geosolutionsgroup.com/en/latest/developer-guide/database-setup/#database-creation-mode) set to `update`.
So if you want to see this information, even if it is not strictily required, you will have to run the migration scripts anyway.

- For script reference see:

    [postgresql migration script 2.0.0 to 2.1.0](https://github.com/geosolutions-it/geostore/blob/master/doc/sql/migration/postgresql/postgresql-migration-from-v.2.0.0-to-v2.1.0.sql)

    [h2 migration script 2.0.0 to 2.1.0](https://github.com/geosolutions-it/geostore/blob/master/doc/sql/migration/h2/h2-migration-from-v.2.0.0-to-v2.1.0.sql)

    [oracle migration script 2.0.0 to 2.1.0](https://github.com/geosolutions-it/geostore/blob/master/doc/sql/migration/oracle/oracle-migration-from-v.2.0.0-to-v2.1.0.sql)

```

File: docs/developer-guide/mapstore-migration-guide.md [hunk 2]
```diff
@@ -22,6 +22,21 @@ This is a list of things to check if you want to update from a previous version
...
+So if you want to see this information, even if it is not strictily required, you will have to run the migration scripts anyway.
+
+- For script reference see:
+
+    [postgresql migration script 2.0.0 to 2.1.0](https://github.com/geosolutions-it/geostore/blob/master/doc/sql/migration/postgresql/postgresql-migration-from-v.2.0.0-to-v2.1.0.sql)
+
+    [h2 migration script 2.0.0 to 2.1.0](https://github.com/geosolutions-it/geostore/blob/master/doc/sql/migration/h2/h2-migration-from-v.2.0.0-to-v2.1.0.sql)
+
+    [oracle migration script 2.0.0 to 2.1.0](https://github.com/geosolutions-it/geostore/blob/master/doc/sql/migration/oracle/oracle-migration-from-v.2.0.0-to-v2.1.0.sql)
+
```
Comment (offtherailz) [inline docs/developer-guide/mapstore-migration-guide.md:38]:
```suggestion
```

**Pull request merged**